tensorrt_llm/serve/generator_ipc.py

```
@@ -0,0 +1,1016 @@
+    audio_sample_rate: Optional[int]
+    error: Optional[str]
+    metrics: Any
+
+
+def _snapshot_visual_output(output: Any) -> VisualGenOutputSnapshot:
+    import torch
+
+    media = {name: getattr(output, name, None) for name in ("image", "video", "audio")}
+    payload_path = None
+    if any(value is not None for value in media.values()):
+        storage_path = Path(os.getenv("TRTLLM_MEDIA_STORAGE_PATH", "/tmp/trtllm_generated"))  # nosec B108
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/trtllm_generated"

(S108)